contracts/exchange/types/DataStructure.sol

```diff
@@ -124,6 +124,10 @@ struct State {
   address initializeConfigSigner;
   // uint configVersion
   uint configVersion;
+  // Total spot balances for all accounts
+  mapping(Currency => int64) totalSpotBalances;
+  // Bridging partners
```

Comment:
add a comment that the number of bridging partners will be < 10 (so that auditor doesn't raise concern about the double for loop perf in one of the assertion)

Reply:
added